osidb/models.py

```diff
@@ -2088,15 +2088,21 @@ def is_community(self) -> bool:
             name=self.ps_module, ps_product__business_unit="Community"
         ).exists()
 
+    # TODO unlike SFM2 this is mostly unused. Is that intentional or did something get missed?
```

Comment:
@osoukup Unlike SFM2, is_compliance_priority is mostly unused in OSIDB. Is that intentional or did something get missed?

(This is sth I identified during work on stream preselection for fedramp, but should be filed as a separate Jira if a valid concern)

Reply:
Yes. Made a comment so that whoever takes it takes it into account.